### PR TITLE
Dataset correction and improvements

### PR DESCRIPTION
**Pull request by Enmartz**

### CODE REVIEW

**Comment by leonsuarez24** [comment]
> _ No description provided. _

**Comment by Enmartz** [comment]
> > _ No description provided. _
> 
> You didn't provide a description

This issue is related to issue #173, as mentioned in the conversation history. Please check it out to review the changes and improvements I made.

**Review by leonsuarez24 — CHANGES_REQUESTED** [review]
Thanks @Enmartz , the datasets are working perfectly.


To further improve the Datasets Module, I suggest updating the documentation to clearly state that the train dataset can be accessed using the train keyword argument. By default, the module uses the validation set for training, and users would need to examine the code to understand how to switch to the training set. Providing this information in both the function documentation and the documentation page would prevent users from having to look into the code.

image

**Comment by Enmartz** [comment]
> Thanks @Enmartz , the datasets are working perfectly.
> 
> To further improve the Datasets Module, I suggest updating the documentation to clearly state that the train dataset can be accessed using the train keyword argument. By default, the module uses the validation set for training, and users would need to examine the code to understand how to switch to the training set. Providing this information in both the function documentation and the documentation page would prevent users from having to look into the code.
> 
> image

Thanks for the improvement advice. I implemented it by removing `**kwargs` and explicitly defining the parameters `train` and `download`. I also modified the rest of the necessary code to adapt this improvement. Please check it out.

**Review by leonsuarez24 — APPROVED** [review]
LGTM
